docs/content/users/quickstart.md

```diff
@@ -404,7 +389,7 @@ The Laravel project type can be used for [Lumen](https://lumen.laravel.com/) lik
 ## Moodle
 
 ```bash
-ddev config --composer-root=public --docroot=public --webserver-type=apache-fpm --database=mariadb:10.6
+ddev config --composer-root=public --docroot=public --webserver-type=apache-fpm
```

Comment:
hm i've retested right now. I must have mixed something up yesterday. i was testing all the sites in the same directory. `ddev delete --omit-snapshot` and after that deleting the entire content of the directory and wipe it clean. i must have missed something so i ran into this error 
```
$> ddev config --composer-root=public --docroot=public --webserver-type=apache-fpm --database=mariadb:10.6
Creating a new DDEV project config in the current directory (/Users/rkoller/Sites/validate) 
Once completed, your configuration will be written to /Users/rkoller/Sites/validate/.ddev/config.yaml
 
Created docroot directory at /Users/rkoller/Sites/validate/public 
Configuring a 'php' project with docroot 'public' at /Users/rkoller/Sites/validate/public 
unable to configure project validate with database type mariadb:10.11 because that database type does not match the current actual database. Please change your database type back to mariadb:10.11 and start again, export, delete, and then change configuration and start. To get back to existing type use 'ddev config --database=mariadb:10.11', and you can try a migration with 'ddev debug migrate-database mariadb:10.6' see docs at https://ddev.readthedocs.io/en/stable/users/extend/database-types/ 
```
that was the reason why i simply gave it a try in combination with the fact that the moodle docs have 10.6 as the minimum requirement while the recommended version is the latest (https://moodledev.io/general/releases/4.3). I've retested now, the install works for both mariadb 10.6 and 10.11. i just thought 10.6 was a hard requirement for whatever reasons by someone who is more familiar with moodle. but from my point of view in combination with what the docs say it is totally fine to go with the latest mariadb version aka 10.11 and therefore it is fine keeping `--database=mariadb:10.6` removed.

Reply:
Thanks, please use new clean directories when testing all the things :)

Reply:
usually i do that. but they were clean anyway (some oversight was the only way i could explain myself that error). but as i said i did a `ddev delete` on the project and then `rm` every file and folder inside the root. it was definitely clean.